aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -140,39 +146,39 @@ struct Vec256 {
 template <class T> Vec256<T> operator+(const Vec256<T> &a, const Vec256<T> &b) {
   Vec256<T> c = Vec256<T>();
   for (int i = 0; i != Vec256<T>::size; i++) {
-    c.values[i] = a.values[i] + b.values[i];
+    c.set_value(i, a[i] + b[i]);
```

Comment:
.values is not indexable for packed types. That means, something like int64_t does not have a definition of say max. It will then use the default implementation. However, __m256i can't be indexed like this and it'll segfault.